client/karma.conf.js

```diff
@@ -0,0 +1,83 @@
+const path = require('path');
```

Comment:
So why the `karma.conf.js` is a better name for a config file (than `karma.config.js`) ?